app/processing/account-codes/get-codes-for-line.js

```diff
@@ -0,0 +1,13 @@
+const { CS } = require('../../constants/schemes')
+const { isCapital } = require('../is-capital')
+
+const getCodesForLine = (schemeId, lineCode, schemeCode, accountCodeMap) => {
+  if (schemeId === CS) {
+    return accountCodeMap.find(x => x.lineCode === lineCode && (isCapital(schemeCode) ? x.capital : x.revenue))
+  }
```

Comment:
For CS, we also need to determine whether it's capital or revenue, can't just rely on the line code.

Some line code have a different domestic account for capital vs revenue